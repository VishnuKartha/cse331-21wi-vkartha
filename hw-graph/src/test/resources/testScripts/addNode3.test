# Creates 2 graphs and adds a node to both of them

# Create Graphs
CreateGraph graph1
CreateGraph graph2

# Add a pair of nodes
AddNode graph1 n1
AddNode graph2 n2