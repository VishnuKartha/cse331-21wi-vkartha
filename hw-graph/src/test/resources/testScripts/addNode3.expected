# Creates 2 graphs and adds a node to both of them

# Create Graphs
created graph graph1
created graph graph2

# Add a pair of nodes
added node n1 to graph1
added node n2 to graph2